### ISSUE TITLE
Export requires graphviz

### ISSUE THREAD

**Issue by mochic** [AllenNeuralDynamics/aind-behavior-curriculum#35]
For some operating systems graphviz may not already be installed. It might be helpful for future users to have a note that they need that in the README.

**Comment by bruno-f-cruz** [comment]
I think the core functionality should be independent of graphviz for sure (this was raised a few times during development). I do agree that this dependency is not explicitly stated tho. How would you feel about adding a new dependency tag to the project.toml file that includes erdantic and graphviz, and try to catch errors when calling methods that expect this lib to be installed? Would this suffice?

**Comment by mochic** [comment]
Yeah this sounds good. Exporting diagrams is an amazing feature. We can still use most functionality of the library by validating via just the base model validation. Any sort of documentation on why the export fails for this reason seems useful, raising specific errors makes it easier on end users I suppose, I just don't want to bog you down with these kinda smallish issues.

**Comment by jwong-nd** [comment]
Looks promising, I can try a build on a separate branch.
